pkg/reconciler/inmemorychannel/controller/controller.go

```diff
@@ -69,6 +71,7 @@ func NewController(
 		endpointsLister:      endpointsInformer.Lister(),
 		serviceAccountLister: serviceAccountInformer.Lister(),
 		roleBindingLister:    roleBindingInformer.Lister(),
+		secretLister:         secretInformer.Lister(),
```

Comment:
In addition, we need to setup an event handler to "global resync" IMCs when the specific secret change, similar to the deployment down below but on the secretInformer:
```golang 
	deploymentInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
		FilterFunc: controller.FilterWithName(dispatcherName),
		Handler:    controller.HandleAll(grCh),
	})
```

This will allow us showing new CA certs when they are rotated